docs/source/io_formats/nuclear_data.rst

```diff
@@ -0,0 +1,353 @@
+.. _usersguide_nuclear_data:
+
+========================
+Nuclear Data File Format
+========================
+
+---------------------
+Incident Neutron Data
+---------------------
+
+
+**/<nuclide name>/**
+
+:Attributes: - **Z** (*int*) -- Atomic number
+             - **A** (*int*) -- Mass number
+             - **metastable** (*int*) -- Metastable state
+             - **atomic_weight_ratio** (*double*) -- Mass in units of neutron masses
+             - **temperature** (*double*) -- Temperature in MeV
+             - **n_reaction** (*int*) -- Number of reactions
```

Comment:
I would prefer that we don't use variables like `n_reaction` and `n_product` since we can get that same information from `data.keys()`, and it would eliminate the possibility of having the wrong value for `n_reaction`.